azure-simple/packer/README.md

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
@rasanjaya85 do we need this extra line?

Reply:
Done